plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/executor/TesFileCopyStrategy.groovy

```diff
@@ -113,7 +113,8 @@ class TesFileCopyStrategy implements ScriptFileCopyStrategy {
             copy.remove('PATH')
         // when a remote bin directory is provide managed it properly
         if( remoteBinDir ) {
-            result << "cp -r ${remoteBinDir}/* \$PWD/nextflow-bin/\n"
+            result << "mkdir -p \$PWD/nextflow-bin/\n"
+            result << "cp -r \$PWD/bin/* \$PWD/nextflow-bin/\n"
```

Comment:
This is the probably biggest change and should be revised as it changes existing behavior of the `bin` $PATH by moving the `bin` filepath from `${remoteBinDir}/dir` to the working directory of the process.